backend/prompt_studio/prompt_studio_output_manager/models.py

```diff
@@ -21,6 +21,9 @@ class PromptStudioOutputManager(BaseModel):
     output = models.CharField(
         db_comment="Field to store output", editable=True, null=True, blank=True
     )
+    context = models.CharField(
+        db_comment="Field to store chucks used", editable=True, null=True, blank=True
```

Comment:
```suggestion
        db_comment="Field to store chunks used", editable=True, null=True, blank=True
```
Typo, please regenerate the migration as well.
@jagadeeswaran-zipstack will this 
1. include the system prompt as well which was sent to the LLM? Or is it just the customer data?
1. Is it okay from a compliance perspective to store this data in our DB like this?
1. Does `CharField` have any size related restrictions?
1. This is already stored by the vector DB - can we map it somehow with IDs maybe? (Can be taken up later)

Reply:
I think @Deepak-Kesavan is the right person to answer. Yes CharField have size restrictions 255 character. Using TextField will be more appropriate here I guess.